modules/badge/react/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@workday/canvas-kit-react-badge",
+  "version": "0.0.0",
+  "description": "provides a summary indicator with dynamic values",
+  "author": "Workday, Inc. (https://www.workday.com)",
+  "license": "Apache-2.0",
+  "main": "dist/commonjs/index.js",
+  "module": "dist/es6/index.js",
+  "sideEffects": false,
+  "types": "dist/es6/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Workday/canvas-kit/tree/master/modules/badge/react"
+  },
+  "files": [
+    "dist/",
+    "lib/",
+    "index.ts",
+    "ts3.5/**/*"
+  ],
+  "typesVersions": {
```

Comment:
This is what is causing builds to fail. I'm not picking up on anything that actually needs to be down-leveled so I'm not sure of the history here.

Reply:
yeah, that config plus `"build:downlevel-dts": "yarn run downlevel-dts dist ts3.5/dist",` was generated by the `create-component` script. I removed the build script, but I missed this. Good catch. 👍

Reply:
@lychyi I added it to all components when we upgraded to TS3.8 in #533

Reply:
@anicholls @alanbsmith Sorry, should have been clearer. What I meant to say was "...as it pertains to this PR". There must have been some history to generating a component using `downlevel-dts` on this PR. TS3.8 is for the next major version right?

Anyway, no big deal, ^ that was what I was really trying to ask. Should have put more context behind it.